sreejasuresh007-P1.md

```diff
@@ -0,0 +1,182 @@
+# Loading data from csv file into postgresql using python
```

Comment:
move the article to articles folder, Because only the articles available in the article folder will be published and other won't be rendered.

Reply:
Done